lib/LANraragi/Utils/Constants.pm

```diff
@@ -0,0 +1,20 @@
+package LANraragi::Utils::Constants;
+
+use strict;
+use warnings;
+use utf8;
+no warnings 'experimental';
+
+use Exporter 'import';
+our @EXPORT_OK = qw(%TANK_METADATA);
+
+
+our %TANK_METADATA = (
+        "name", 0,
+        "summary", -1,
+        "thumbhash", -2,
```

Comment:
Using negative indices for the metadata is clever btw, I hadn't thought of that!  

My opinion might be changed on this, but I don't think thumbhash and alias are needed here:  

- Since you can't execute plugins on tanks there's no need to store a thumb hash.  
  - But we might need another thumbnail-related field here in the future; I'm not 100% on what the story is for tank thumbnails yet: Should they just use the thumbnail of the first archive? A composite of all archive thumbs done thru magick? An arbitrary image path? I'll probably keep it simple for now and just use the first archive thumb  
- What is alias even meant to be?  

Tags should always just be the aggregate of all archive tags, but one could argue a field here would be useful for caching.. I was thinking about that while doing the search implementation so might as well keep it for now.

Reply:
I mean, the frontend doesn't use thumbhash in any form or fashion, that's just used by plugins for reverse image searching.  
Thumbnail lookup only needs an archive ID, which is why I was planning to only use the first archive ID for now.  

As for alias, I think we could add it later if needed, yeah.